.readthedocs.yaml

```
@@ -6,9 +6,8 @@ build:
     python: "3.12"
   jobs:
     post_install:
-      - pip install uv
-      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH uv pip install .[doc]
+      - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH pip install .[doc]
 sphinx:
   configuration: docs/conf.py
   # disable this for more lenient docs builds
-  fail_on_warning: true
+  fail_on_warning: false
```

I really strongly advise to leave that on `true`

```suggestion
  fail_on_warning: true
```

---

yeah I'm aware of the comment on scanpy regarding this setting ;)
I was only checking if it helps (it doesnt), but don't intend to keep this